DIRECTORY.md

```diff
@@ -51,6 +51,15 @@
     * [Segment Tree](https://github.com/TheAlgorithms/C/blob/HEAD/data_structures/binary_trees/segment_tree.c)
     * [Threaded Binary Trees](https://github.com/TheAlgorithms/C/blob/HEAD/data_structures/binary_trees/threaded_binary_trees.c)
     * [Words Alphabetical](https://github.com/TheAlgorithms/C/blob/HEAD/data_structures/binary_trees/words_alphabetical.c)
+  * Bloom Filters
+    * Bloom Filter
+      * [Bloom Filter](https://github.com/TheAlgorithms/C/blob/HEAD/data_structures/bloom_filters/bloom_filter/bloom_filter.c)
+      * [Bloom Filter](https://github.com/TheAlgorithms/C/blob/HEAD/data_structures/bloom_filters/bloom_filter/bloom_filter.h)
+      * [Main](https://github.com/TheAlgorithms/C/blob/HEAD/data_structures/bloom_filters/bloom_filter/main.c)
+    * Counting Bloom Filter
+      * [Counting Bloom Filter](https://github.com/TheAlgorithms/C/blob/HEAD/data_structures/bloom_filters/counting_bloom_filter/counting_bloom_filter.c)
+      * [Counting Bloom Filter](https://github.com/TheAlgorithms/C/blob/HEAD/data_structures/bloom_filters/counting_bloom_filter/counting_bloom_filter.h)
+      * [Main](https://github.com/TheAlgorithms/C/blob/HEAD/data_structures/bloom_filters/counting_bloom_filter/main.c)
```

Comment:
It's not hard to see the way you organize your code messed up the auto-generated directory. Condense your code into two files, one for regular, one for counting to fix this.